usaspending_api/api_contracts/contracts/v2/idvs/count/federal_account/award_id.md

```diff
@@ -0,0 +1,50 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Federal Accounts Count [/api/v2/idvs/count/federal_account/{award_id}/]
+
+This endpoint is used for the federal accounts tab on the idv summary page.
+
+## GET
+
+This endpoint returns the number of federal accounts associated with the given IDV and it's children and grandchildren.
+
++ Request (application/json)
+    A request with a award (contract or assistance) id 
+    + Parameters        
+        + `award_id`: `CONT_IDV_NNK14MA74C_8000` (required, string)
+            Either a "generated" natural award id (string) or a database surrogate award id (number).  Generated award identifiers are preferred as they are effectively permanent.  Surrogate award ids are retained for backward compatibility but are deprecated.
+
++ Response 200 (application/json)
+    + Attributes 
+        + `count` (required, number)
+     + Body
+    
+            {
+                "count": 2
+            }
+            
+# Federal Accounts Count Filtered by PIID [/api/v2/idvs/count/federal_account/{award_id}/{?piid}]
```

Comment:
Personally I would just use this second GET to demonstrate the endpoint instead of having both since all of the verbiage is the same except the `piid`. You state that `piid` is `(optional, string)` and so that should be enough to show people that you can have the endpoint with or without.